routing/dht/routing.go

```diff
@@ -28,6 +29,8 @@ var asyncQueryBuffer = 10
 // PutValue adds value corresponding to given Key.
 // This is the top level "Store" operation of the DHT
 func (dht *IpfsDHT) PutValue(ctx context.Context, key key.Key, value []byte) error {
+	ctx, cancel := context.WithTimeout(ctx, time.Second*10) // CR: should this be removed instead?
```

Comment:
Does this mean all dht ops or some dht ops (just PutValue, GetValue, etc)?

With the same constant? Longest RTT is not more than 1s (http://www.richardclegg.org/sites/default/files/papers/ifip_nets_2012.pdf).
So operation like `FindPeer` doesn't require a 60s timeout.